resources/docs/source/includes/_structures/beatmap_scores.md

```diff
@@ -0,0 +1,17 @@
+## BeatmapScores
+```json
+{
+  "scores": [],
+  "userScore": {
+    "position": 1,
+    "score": {}
+  }
+}
+```
+
+Field              | Type              | Description
+------------------ | ----------------- | --------------------------------------------------------------------
+scores             | [Score](#score)[] | The list of top scores for the beatmap in descending order.
+userScore          | Object?           | The score of the current user. This is not returned if the current user does not have a score.
+userScore.position | number            | The position of the user's score in the Beatmap's ranking, if available.
```

Comment:
the whole `userScore` can probably be updated to just contain `json_item` of current user score with `rank_global` and `rank_country` includes... except we'll have to add support for passing mods for the transformer first. (also `rank_friend`. or maybe a whole new `rank` include which also returns which mod/mode it applies to)